### PR TITLE
[docs][yba][DOC-47][DOC-48] Changing replication factor of read replicas

### PR DESCRIPTION
**Pull request by ddhodge**
Update instructions for changing RF of read replicas
DOC-47

@netlify /preview/yugabyte-platform/create-deployments/read-replicas/

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/create-deployments/read-replicas.md [hunk 1]
```diff
@@ -11,42 +11,35 @@ menu:
...
 To finish the process, click **Create**.
 
-## Examine the universe
-
-While the universe is being created, **Dashboard** should look similar to the following illustration:
-
-![Universe Waiting to Create](/images/ee/universe-waiting.png)
-
-Once the universe has been created, **Dashboard** displays the primary and read replica cluster information, as well as shows the distinct clusters on the map.
+After the universe has been created, **Dashboard** displays the primary and read replica cluster information, as well as shows the distinct clusters on the map.
```
Comment (rajmaddy89) [inline docs/content/preview/yugabyte-platform/create-deployments/read-replicas.md:42]: Dashboard usually displays only Primary Cluster information, I agree that map display distinct cluster info though

File: docs/content/stable/yugabyte-platform/create-deployments/read-replicas.md [hunk 2]
```diff
@@ -25,28 +29,17 @@ You start by navigating to **Dashboard** and clicking **Create Universe**. Use t
...
 To finish the process, click **Create**.
 
-## Examine the universe
-
-While the universe is being created, **Dashboard** should look similar to the following illustration:
-
-![Universe Waiting to Create](/images/ee/universe-waiting.png)
-
-Once the universe has been created, **Dashboard** displays the primary and read replica cluster information, as well as shows the distinct clusters on the map.
+After the universe has been created, **Dashboard** displays the primary and read replica cluster information, as well as shows the distinct clusters on the map.
```
Comment (rajmaddy89) [inline docs/content/stable/yugabyte-platform/create-deployments/read-replicas.md:42]: Same as above

File: docs/content/preview/yugabyte-platform/create-deployments/read-replicas.md (hunk 1 above)
Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/create-deployments/read-replicas.md:42]: Think we can just drop it really

File: docs/content/stable/yugabyte-platform/create-deployments/read-replicas.md [hunk 3]
```diff
@@ -11,77 +11,57 @@ menu:
...
-While the universe is being created, **Dashboard** should look similar to the following illustration:
+    {{< /tip >}}
 
-![Universe Waiting to Create](/images/ee/universe-waiting.png)
+1. Click **Configure Read Replica**.
+1. Specify the following on the **Read Replica** tab to create a read replica cluster:
 
-Once the universe has been created, **Dashboard** displays the primary and read replica cluster information, as well as shows the distinct clusters on the map.
+    - Specify the regions where you want to place replicas.
+    - Specify the number of nodes and the number of read replicas. The number of nodes must exceed the number of replicas.
```
Comment (rajmaddy89) [inline docs/content/stable/yugabyte-platform/create-deployments/read-replicas.md:39]: The number of nodes must be equal or greater than the number of replicas.

File: docs/content/preview/yugabyte-platform/create-deployments/read-replicas.md [hunk 4]
```diff
@@ -11,77 +11,57 @@ menu:
...
-While the universe is being created, **Dashboard** should look similar to the following illustration:
+    {{< /tip >}}
 
-![Universe Waiting to Create](/images/ee/universe-waiting.png)
+1. Click **Configure Read Replica**.
+1. Specify the following on the **Read Replica** tab to create a read replica cluster:
 
-Once the universe has been created, **Dashboard** displays the primary and read replica cluster information, as well as shows the distinct clusters on the map.
+    - Specify the regions where you want to place replicas.
+    - Specify the number of nodes and the number of read replicas. The number of nodes must exceed the number of replicas.
```
Comment (rajmaddy89) [inline docs/content/preview/yugabyte-platform/create-deployments/read-replicas.md:39]: The number of nodes must be equal or greater than the number of replicas.

**Review by rajmaddy89 — APPROVED** [review]
LGTM

**Review by Lingeshwar — APPROVED** [review]
LGTM

**Comment by yushenng** [comment]
See comments in this Slack DM:

https://yugabyte.slack.com/archives/C05ETJKLEQL/p1687988713563329

**Pull request merged**